tests/app/templating/summary/test_question.py

```diff
@@ -243,7 +243,7 @@ def test_question_should_be_skipped(self):
         # Given
         answers = {'answer_1': 'skip me'}
         answer_schema = {'id': 'answer_1', 'title': '', 'type': '', 'label': ''}
-        skip_condition = {'when': {'id': 'answer_1', 'condition': 'equals', 'value': 'skip me'}}
+        skip_condition = {'when': [{'id': 'answer_1', 'condition': 'equals', 'value': 'skip me'}]}
```

Comment:
Fair enough, I'm leaving this for now - the whole set is like this and I'm not about to upset the coverage gods anymore.